samples/quickstart.js

```diff
@@ -1,66 +1,94 @@
+// Copyright 2020 Google LLC
+//
 // Licensed under the Apache License, Version 2.0 (the "License");
 // you may not use this file except in compliance with the License.
 // You may obtain a copy of the License at
 //
-//     https://www.apache.org/licenses/LICENSE-2.0
+//      http://www.apache.org/licenses/LICENSE-2.0
 //
 // Unless required by applicable law or agreed to in writing, software
 // distributed under the License is distributed on an "AS IS" BASIS,
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-//
 
 'use strict';
 
-async function main(
-  projectId,
-  propertyId,
-  startDate,
-  endDate,
-  nameOfDimensions,
-  nameOfMetrics
-) {
-  // [START cloud_data_analytics_quickstart]
-  // Imports the Google Cloud client library
-
-  // eslint-disable-next-line node/no-extraneous-require, node/no-missing-require
-  const {AlphaAnalyticsDataClient} = require('@google-analytics/data');
+/** This application demonstrates the usage of the Analytics Data API using
+ service account credentials. For more information on service accounts, see
+
+ https://cloud.google.com/iam/docs/understanding-service-accounts
+
+ The following document provides instructions on setting service account
+ credentials for your application:
+
+ https://cloud.google.com/docs/authentication/production
+
+ In a nutshell, you need to:
+
+ 1. Create a service account and download the key JSON file.
+ https://cloud.google.com/docs/authentication/production#creating_a_service_account
 
-  // TODO(developer): replace with your prefered project ID.
-  // const projectId = 'my-project'
+ 2. Provide service account credentials using one of the following options:
+ - set the GOOGLE_APPLICATION_CREDENTIALS environment variable, the API
+ client will use the value of this variable to find the service account key
+ JSON file.
+
+ https://cloud.google.com/docs/authentication/production#setting_the_environment_variable
+
+ OR
+ - manually pass the path to the service account key JSON file to the API client
+ by specifying the keyFilename parameter in the constructor.
+
+ https://cloud.google.com/docs/authentication/production#passing_the_path_to_the_service_account_key_in_code
+ */
+
+function main(propertyId = 'YOUR-GA4-PROPERTY-ID') {
```

Comment:
sorry, let's also add projectId to this list of variables, and then use it when we're calling the function in the test. We'll also want to comment out the project Id in the list of variables under calling the main function (see #6 in the guidelines I sent over!)